pymeasure/instruments/instrument.py

```diff
@@ -46,7 +47,7 @@ class Instrument(object):
     """
 
     # noinspection PyPep8Naming
-    def __init__(self, adapter, name, includeSCPI=True, **kwargs):
+    def __init__(self, adapter, name, includeSCPI=True, command_delay=None, **kwargs):
```

Comment:
The new parameter will need an entry in the class docstring. Please ensure that you explain which commands will be delayed.

Reply:
Yes, will do!